### DISCUSSION TITLE
Item swaps for weapons

### DISCUSSION THREAD

**Discussion by CozmicDev** [xivdev/Penumbra#334] [Ideas] [answered]
Would be nice if we had the option to use the Advanced Editing > Item swap, and have so we can swap weapons/shields
![image_2023-09-04_090232036](https://github.com/xivdev/Penumbra/assets/93140062/61f9651f-805d-456e-bc2e-f7a680debc72)

**Comment by SerenitySublime** [comment]
Dev already stated that weapon item swap will never be a thing as there are too many reasons why things could go bad or that it wouldn't work.

**Reply by CozmicDev** [reply]
Couldn't find anything about it earlier, thank you

**Comment by Theo-Asterio** [comment, marked as answer]
This isn't possible because of how weapons work. Different Skeletons, Animations, Binding Points and more past that make it not possible. Textools doesn't allow for this properly with item swap without breaking things tremendously, Penumbra will not allow you to break things yourself.

Please use Glamourer if you want to swap what weapon you're using that matches your class.

**Reply by CozmicDev** [reply]
Ahh makes sense, tysm
